Sources/SCTE35/ModelObjects/SpliceDescriptors/SegmentationInfo.swift

```diff
@@ -265,12 +265,12 @@ public struct PIDComponent {
 }
 
 /// Segmentation_upid according to segmentation_upid_type as defined in Table 21.
-public struct SegmentationUPID {
+public struct SegmentationUPID: Equatable {
     /// A value from the following table. There are multiple types allowed to ensure that programmers will be able to use an id that their systems support. It is expected that the consumers of these ids will have an out-of-band method of collecting other data related to these numbers and therefore they do not need to be of identical types. These ids may be in other descriptors in the program and, where the same identifier is used (ISAN for example), it shall match between programs.
-    public let type: Int
+    public let type: UInt8
```

Comment:
This property is defined in the spec to be exaclty 8 bits.  Therefore it made sense to me to constrain the integer type to this.